_includes/manuals/1.24/3.2.4_tuning_options.md

```diff
@@ -0,0 +1,54 @@
+The Foreman installer supports automatic tuning of your environment using predefined tuning profiles.  These tuning profiles are the result of a culmination of extensive learning from foreman environments deployed at scale in large customer environments.
+
+When the foreman-installer is run, it is deployed with a `default` predefined tuning profile. The definition of this profile can be found in `/usr/share/foreman-installer/config/foreman.hiera/tuning/common.yaml`.
```

Comment:
Might reword to `with a predefined tuning profile named default` to make it clearer.

Reply:
Consider changing `this` to `the default profile` to make it clearer